src/PersonaWithCard.test.tsx

```diff
@@ -4,7 +4,7 @@ import { buildProfile } from "./supporting/Profile";
 import { PersonaShowMode, IPersonaProfile } from "./Types";
 import { GraphService } from "./GraphService";
 import { waitNextTick } from "./supporting/Promise";
-import { PersonaSize, HoverCard } from "@fluentui/react";
+import { PersonaSize, HoverCard} from "@fluentui/react";
```

Comment:
Nit: add back whitespace that was deleted.
Also, super-nit: can you put the imports in alphabetical order here (i.e. HoverCard, then PersonaSize)?

Reply:
Consider adding prettier and the space 1es lint rule to this project, then adding it as a PR build step? https://github.com/microsoft-1es/eslint-config-1es-space